pkg/cloudprovider/providers/gce/gce_cert.go

```diff
@@ -29,46 +33,33 @@ func newCertMetricContext(request string) *metricContext {
 // GetSslCertificate returns the SslCertificate by name.
 func (gce *GCECloud) GetSslCertificate(name string) (*compute.SslCertificate, error) {
 	mc := newCertMetricContext("get")
-	v, err := gce.service.SslCertificates.Get(gce.projectID, name).Do()
+	v, err := gce.c.SslCertificates().Get(context.Background(), meta.GlobalKey(name))
 	return v, mc.Observe(err)
 }
 
 // CreateSslCertificate creates and returns a SslCertificate.
 func (gce *GCECloud) CreateSslCertificate(sslCerts *compute.SslCertificate) (*compute.SslCertificate, error) {
 	mc := newCertMetricContext("create")
-	op, err := gce.service.SslCertificates.Insert(gce.projectID, sslCerts).Do()
-
+	err := gce.c.SslCertificates().Insert(context.Background(), meta.GlobalKey(sslCerts.Name), sslCerts)
 	if err != nil {
 		return nil, mc.Observe(err)
 	}
-
-	if err = gce.waitForGlobalOp(op, mc); err != nil {
-		return nil, mc.Observe(err)
-	}
-
 	return gce.GetSslCertificate(sslCerts.Name)
 }
 
 // DeleteSslCertificate deletes the SslCertificate by name.
 func (gce *GCECloud) DeleteSslCertificate(name string) error {
 	mc := newCertMetricContext("delete")
-	op, err := gce.service.SslCertificates.Delete(gce.projectID, name).Do()
-
-	if err != nil {
-		if isHTTPErrorCode(err, http.StatusNotFound) {
-			return nil
-		}
-
-		return mc.Observe(err)
+	err := gce.c.SslCertificates().Delete(context.Background(), meta.GlobalKey(name))
+	if isHTTPErrorCode(err, http.StatusNotFound) {
+		err = nil
```

Comment:
Don't think we need to return nil for NotFound anymore. Both ingress-gce and the e2e/ingress-utils check for NotFound.

Reply:
done